src/models/Presentation.js

```diff
@@ -3,4 +3,5 @@ import type { Image } from './Image';
 
 export type Presentation = {
   images: Array<Image>,
+  searching: boolean,
```

Comment:
Why?

Reply:
Because as soon as you press search, you get redirected to /presentation?search=whatever.
But the images are still sent via socketio, so we need to show the loading icons until we get them from socketio. We cannot do that with local state, because that would break if we reload the presentation page.